askama_derive/src/generator.rs

```diff
@@ -73,9 +90,73 @@ fn build_template(ast: &syn::DeriveInput) -> Result<String, CompileError> {
     if input.print == Print::Code || input.print == Print::All {
         eprintln!("{code}");
     }
+
+    #[cfg(feature = "cache")]
+    cache_template(&digest, &code).unwrap();
+
     Ok(code)
 }
 
+#[cfg(feature = "cache")]
+fn find_cached_template(digest: &str) -> Option<String> {
+    let root = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+    let cache_dir = root.join(".askama");
+    let path = cache_dir.join(digest);
+
```

Comment:
I don't think we should `unwrap()` here. We can just return `None` instead so that everything will still work. Can write the tail expression something like `path.exists().then(|| fs::read_to_string(path).ok()).flatten()`.